persistence/src/commonMain/db_user/com/wire/kalium/persistence/Calls.sq

```diff
@@ -10,12 +10,13 @@ CREATE TABLE Call (
     conversation_type TEXT AS ConversationEntity.Type NOT NULL,
     created_at TEXT NOT NULL,
     type TEXT AS CallEntity.Type NOT NULL DEFAULT 'UNKNOWN',
-    PRIMARY KEY (conversation_id, id)
+    PRIMARY KEY (id, conversation_id)
 );
+
 CREATE INDEX call_date_index ON Call(created_at);
-CREATE INDEX call_conversation_index ON Call(conversation_id);
```

Comment:
with the pk being (conversation_id, id) this index was doing nothing
using the ID first in the PK since it is needed in the view and setting the new index to (conversation_id, created_at DESC) since this is how it is being queried